api/db/README.md

```diff
@@ -0,0 +1,52 @@
+# PostgreSQL
+
+We use PostgreSQL 13.x to store data for CompE+.
+
+# Local Development
+
+To run a Postgres instance locally we leverage Docker. The following instructions are based on [this](https://hackernoon.com/dont-install-postgres-docker-pull-postgres-bee20e200198).
+
+## Setup
+
+1. [Install Docker](https://www.docker.com/get-started)
+2. Pull down the Postgres Docker image `docker pull postgres:13`
+3. Create a directory to serve as the local host mount point for Postgres data files `mkdir -p $HOME/docker/volumes/postgres`
+4. Install `psql` via `sudo apt-get install postgresql-client`
+
+## Starting Postgres
+
+`docker run --rm --name pg-docker -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -e POSTGRES_DB=compe-plus -d -p 5432:5432 -v $HOME/docker/volumes/postgres:/var/lib/postgresql/data postgres:13`
+
+Verify that it is running with `docker ps` or kill it with `docker kill pg-docker`
+
+## Connecting to Postgres
+
+`psql -h localhost -U postgres -d compe-plus` (Password is "postgres")
```

Comment:
Does this imply we need to have postgres installed in our machines or should this run under something like `docker exec ...` to the container we ran in the previous step?

Reply:
I callout installing `psql` earlier in the README with `sudo apt-get install postgresql-client`. This installs `psql` and not all of postgres.

Reply:
Ah my bad I missed that. I'm assuming for Mac and Windows they'll have to get a postgresql client container?